toolkit/templates/instruction-list.html

```diff
@@ -64,20 +64,36 @@ <h2 class="instruction-list-item-body">{{ item.body }}</h2>
             {% endwith %}
           {% endfor %}
         {% endif %}
+        {% if item.content %}
+          {% for item_content in item.content %}
```

Comment:
I'm not sure this is any better, sorry! - you've still overloaded the word 'content' to sometimes be a dictionary, and sometimes be a list.

I think it might be a lot simpler (assuming we keep this template) if item.content was just a blob of markup that got passed in, rather than having list items with various types at all? It'd be a lot more flexible if we want to have various things in each step in future.

I get that along with the types there are a bunch of classes  i.e. instruction-list-item-top etc.

Is there actually CSS defined for those in the frontend-toolkit?